Rubberduck.Core/UI/CodeExplorer/Commands/AnnotateDeclarationCommand.cs

```diff
@@ -0,0 +1,161 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Navigation.CodeExplorer;
+using Rubberduck.Parsing.Annotations;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Refactorings;
+using Rubberduck.Refactorings.AnnotateDeclaration;
+using Rubberduck.Refactorings.Exceptions;
+using Rubberduck.UI.Command.Refactorings.Notifiers;
+using Rubberduck.VBEditor.Events;
+
+namespace Rubberduck.UI.CodeExplorer.Commands
+{
+    public class AnnotateDeclarationCommand : CodeExplorerCommandBase
+    {
+        private static readonly Type[] ApplicableNodes =
+        {
+            typeof(CodeExplorerComponentViewModel),
+            typeof(CodeExplorerMemberViewModel)
+        };
+
+        private readonly RubberduckParserState _state;
+
+        private readonly IRefactoringAction<AnnotateDeclarationModel> _annotateAction;
+        private readonly IRefactoringFailureNotifier _failureNotifier;
+        private readonly IRefactoringUserInteraction<AnnotateDeclarationModel> _userInteraction;
+
+        public AnnotateDeclarationCommand(
+            AnnotateDeclarationRefactoringAction annotateAction,
+            AnnotateDeclarationFailedNotifier failureNotifier,
+            RefactoringUserInteraction<IAnnotateDeclarationPresenter, AnnotateDeclarationModel> userInteraction,
+            IVbeEvents vbeEvents, 
+            RubberduckParserState state) 
+            : base(vbeEvents)
+        {
+            _annotateAction = annotateAction;
+            _failureNotifier = failureNotifier;
+            _userInteraction = userInteraction;
+            _state = state;
+
+            AddToCanExecuteEvaluation(SpecialEvaluateCanExecute);
+        }
+
+        public override IEnumerable<Type> ApplicableNodeTypes => new[] { typeof(System.ValueTuple<IAnnotation, ICodeExplorerNode>) };
```

Comment:
This seems to be unused in the diff... it also honestly doesn't seem to make a whole lot of sense to me